engine/src/main/scala/cromwell/engine/io/gcs/GcsPath.scala

```diff
@@ -6,7 +6,7 @@ import scala.util.{Failure, Success, Try}
 /** Represents a Google Cloud Storage path, like gs://bucket/path/to/object.txt
   *
   * @param bucket - should adhere to https://cloud.google.com/storage/docs/bucket-naming?hl=en#requirements
-  * @param objectName
+  * @param objectName - the name of the thing
```

Comment:
yeah, can't win.  The first bit of doc adds real value, but if I don't put **something** here IntelliJ whines.